met-api/migrations/versions/fc570c0faace_survey_migration.py

```diff
@@ -0,0 +1,41 @@
+"""Survey Migration
+
+Revision ID: fc570c0faace
+Revises: d0f92ae9ba77
+Create Date: 2022-07-05 12:21:37.175093
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'fc570c0faace'
+down_revision = 'ec504565fab3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('survey',
+    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+    sa.Column('name', sa.String(length=50), nullable=True),
```

Comment:
we can throw in an index for name and id i think

Reply:
Primary key ix should be sufficient, added ix for name